src/tools/r2rdump/R2RReader.cs

```diff
@@ -226,5 +230,42 @@ private int GetEntryPointIdFromOffset(int offset)
 
             return (int)id;
         }
+
+        /// <summary>
+        /// Returns a formatted string containing a block of bytes from the relative virtual address and size
+        /// </summary>
+        public string DumpBytes(int rva, uint size)
```

Comment:
Design-wise, it would be better to reorganize this code so that the DumpBytes will just return the actual bytes from the image and then whichever class is responsible for writing to the output can take those bytes and format them appropriately.

Reply:
Do we need a function that returns the bytes? This would require copying all the bytes when we could just access it using the address instead. I could move DumpBytes to R2RDump and have it write it out